src/sentry/integrations/slack/command_endpoint.py

```diff
@@ -1,25 +1,38 @@
 import logging
 from typing import Mapping
 
-from django.http import HttpResponse
+from django.http import Http404, HttpResponse
 from rest_framework.request import Request
 
 from sentry.api.base import Endpoint
+from sentry.integrations.slack.link_team import build_linking_url
 from sentry.integrations.slack.message_builder.help import SlackHelpMessageBuilder
 from sentry.integrations.slack.requests.base import SlackRequestError
 from sentry.integrations.slack.requests.command import SlackCommandRequest
+from sentry.models import Identity, IdentityProvider
 
 logger = logging.getLogger("sentry.integrations.slack")
+LINK_TEAM_MESSAGE = "Link your Sentry team to this Slack channel! <{associate_url}|Link your team now> to receive notifications of issues in Sentry in Slack."
+LINK_USER_MESSAGE = "You must first link your identity to Sentry by typing /sentry link. Be aware that you must be an admin or higher in your Sentry organization to link your team."
 
 
 def get_command(payload: Mapping[str, str]) -> str:
-    return payload.get("text", "").split(" ")[0].lower()
+    return payload.get("text", "").lower()
```

Comment:
This should be like command line and return a command string and a list of arguments. So if the user types
```
/sentry link team
```
`"link"` is the command and `["team"]` is the argument.

Reply:
@mgaeta this is what the payload looks like though:

```
{
    'command': '/sentry', 
    'text': 'link team',  
}
```

Reply:
I think if we want to add more commands we'll want to interpret the `text` field as a "command" plus "arguments". For example:
- `/sentry help link`
- `/sentry teams list`
- `/sentry mute alerts deploys`

Reply:
I don't think I did exactly this but given we don't have any lengthy arguments planned I think it's okay for now.